pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[tool.poetry]
+name = "clickhouse-sqlalchemy"
+version = "0.3.2"
+description = "SQLAlchemy dialect for ClickHouse"
+authors = ["Konstantin Lebedev <kostyan.lebedev@gmail.com>"]
+license = "MIT"
+readme = "README.rst"
+include = ["CHANGELOG.md"]
+exclude = ["tests"]
+repository = "https://github.com/xzkostyan/clickhouse-sqlalchemy"
+documentation = "https://clickhouse-sqlalchemy.readthedocs.io/en/latest/"
+keywords = ["clickhouse", "sqlalchemy"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: SQL",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Database",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Application Frameworks",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+]
+
+[tool.project.urls]
+"PyPI" = "https://pypi.org/project/clickhouse-sqlalchemy/"
+"Changes" = "https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/master/CHANGELOG.md"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4.0"
+alembic = "^1.13.3"
```

Comment:
Why dependency versions are fixed?

Reply:
Are they?

Reply:
https://python-poetry.org/docs/dependency-specification/#caret-requirements
This version specification means `alembic>=1.13.3,<1.14`, not `alembic>=1.13.3`. This prevent users from installing newer releases of alembic, like 1.14.0.

At the same time, pdm-based PR #348 uses `alembic>=1.14.0`, with no upper bound:
https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/97aa128a16eda450d6b7cb3b3bce54c9180c73ec/pyproject.toml#L8

And master branch does not include alembic as dependency in the first place:
https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/f52087844b86a9524c2295185ac29274b293d129/setup.py#L98

Reply:
Hm, I cannot recall specifying explicit version when adding packages.

Moreover, I think I have to close this PR in favour of #348 , your help in reviewing or coauthoring is welcome and appreciated.